content/tutorials/reactjs-realtime-commenting.textile

```diff
@@ -22,140 +22,145 @@ tags:
     - Realtime commenting
 ---
 
-In this tutorial, we will see how to use Ably "Realtime client library":/realtime to build a realtime commenting system with React. We'll be making use Ably's public channel. Once a comment is made, we'll publish it to the public channel. We will also be subscribed to the channel so we can see comments as they are added in realtime.
+In this tutorial, we will see how to use Ably "Realtime client library":/realtime to build a realtime commenting system with React. You can see a working demo "here":http://ablycommentingdemo.infinityfreeapp.com/.
+
+We'll be making use of Ably's public channel. Once a comment is made, we'll publish it to the public channel. We will also be subscribed to the channel so we can see comments as they are added in realtime.
 
 This tutorial will not include any server-side implementation. Hence, comments won't be persisted to any storage.
 
 React is a JavaScript library for building user interfaces. It is component-based and declarative which makes it easy to build complex UI with it. For more information on React, please refer to their official website "https://reactjs.org":https://reactjs.org.
 
-<img src="/images/tutorials/react-commenting/react_logo.png" style="width: 50%" alt="React logo">
+<img src="/app/assets/images/tutorials/react-commenting/react_logo.png" style="width: 50%" alt="React logo">
```

Comment:
not sure about the path here.
I will work on the docs repo in the morning so I will double-check this for my own sake.

Reply:
Image was broken. Fixed and also make link open in new window.